includes/class-sensei-data-cleaner.php

```diff
@@ -57,15 +57,15 @@ class Sensei_Data_Cleaner {
 		'sensei_installed',
 		'sensei_course_enrolment_site_salt',
 		'sensei_course_order',
-		'skip_install_sensei_pages',
+		'skip_install_sensei_pages', // @deprecated since 3.1.0.
```

Comment:
Using `// @deprecated 3.1.0.` here triggers PHPCS's *that looks a lot like code to me* warning, so it has to be a little more verbose.

Reply:
I think it's okay here because it's a different situation :)